### PR TITLE
removed unused package

### PR DESCRIPTION
**Pull request by ZandileDube**

### CODE REVIEW

File: src/Account.java
Comment (africamatji) [inline src/Account.java:1]: Why did you remove this line?

Comment (ZandileDube) [reply src/Account.java:1]: the package was unused

Comment (africamatji) [reply src/Account.java:1]: Ok cool, go ahead and merge please.
